From 803fd4e9b127241e06d0616d4b47b03b652a37bc Mon Sep 17 00:00:00 2001
From: Bjoern Michaelsen <bjoern.michaelsen@canonical.com>
Date: Tue, 19 Jul 2011 17:20:43 +0200
Subject: [PATCH 5/5] disable
 --dynamic-list-cpp-new/--dynamic-list-cpp-typeino as
 they SIGSEGV ld

 * at least on linking transex in l10ntools/source with ld from binutils-2.21.52.20110707-1ubuntu1
---
 solenv/gbuild/platform/unxgcc.mk |    2 --
 solenv/inc/unxgcc.mk             |    2 +-
 2 files changed, 1 insertions(+), 3 deletions(-)

diff --git a/solenv/gbuild/platform/unxgcc.mk b/solenv/gbuild/platform/unxgcc.mk
index 7ad122e..2c3d4b5 100755
--- libreoffice-3.5.0/solenv/gbuild/platform/unxgcc.mk
+++ libreoffice-3.5.0/solenv/gbuild/platform/unxgcc.mk
@@ -130,8 +130,6 @@ endif
 
 ifneq ($(HAVE_LD_BSYMBOLIC_FUNCTIONS),)
 gb_LinkTarget_LDFLAGS += \
-	-Wl,--dynamic-list-cpp-new \
-	-Wl,--dynamic-list-cpp-typeinfo \
 	-Wl,-Bsymbolic-functions \
 
 endif
diff --git a/solenv/inc/unxgcc.mk b/solenv/inc/unxgcc.mk
index c73333f..8c19388 100644
--- libreoffice-3.5.0/solenv/inc/unxgcc.mk
+++ libreoffice-3.5.0/solenv/inc/unxgcc.mk
@@ -180,7 +180,7 @@ LINKFLAGSRUNPATH_NONE=
 # flag -Wl,-z,noexecstack sets the NX bit on the stack
 LINKFLAGS=-Wl,-z,noexecstack -Wl,-z,combreloc $(LINKFLAGSDEFS) $(LINKFLAGS_SYSBASE)
 .IF "$(HAVE_LD_BSYMBOLIC_FUNCTIONS)"  == "TRUE"
-LINKFLAGS += -Wl,-Bsymbolic-functions -Wl,--dynamic-list-cpp-new -Wl,--dynamic-list-cpp-typeinfo
+LINKFLAGS += -Wl,-Bsymbolic-functions
 .ENDIF
 
 # linker flags for linking applications
-- 
1.7.5.4
